### ISSUE TITLE
Not prepared for sf2 hitboxes

### ISSUE THREAD

**Issue by iuritorres** [JordanAnthonyKing/sf2-hitbox-extractor#1]
Hello Jordan, I'm recreatring a street fighter 2 game (just 4fun), and I was intending to get all the original's spritesheet frame data, and of what I understood, your script log them in a .txt file, right?

Well, I copied your script and pasted it on "\Fightcade\emulator\fbneo\fbneo-training-mode\hitboxes\st-hitboxes.lua", then I opened my Fightcade 2, installed many different versions of street fighter 2, and follow these steps for all of them:

> TRAINING > Game > Lua Scripting > New Lua Script Window > Edit > selected this script: "C:\Users\pc\Documents\Fightcade\emulator\fbneo\fbneo-training-mode\hitboxes\st-hitboxes.lua" > Run.

And all of them returned this error: not prepared for <sf2> hitboxes...

Can you help me with it?

![image](https://github.com/user-attachments/assets/fda97338-2e37-47cf-87d6-a0c0afdf6d64)

**Comment by JordanAnthonyKing** [comment]
I don't believe you need to load the script manually like that, just place the script file in the location and then launch into training mode normally. It should start outputting as you do different moves. Is your game launched in training mode? I can't tell from the screenshot.

**Comment by iuritorres** [comment]
Yea it was in training mode, I tried different moves but any files has appeared in `emulator\fbneo\fbneo-training-mode` as README says...

![image](https://github.com/user-attachments/assets/556557b5-2505-4682-9e14-a3b4d5b71b0b)
![image](https://github.com/user-attachments/assets/895bb439-c722-4838-9c6f-f1a97b33e7ee)
![image](https://github.com/user-attachments/assets/ae90da8c-b0cb-4985-9ddb-fbd4ad32de66)

Nothing appears :/

I tried instead of editing the `st-hitboxes.lua` file, editing `sf2-hitboxes.lua`, the file `hitboxes.txt` appeared but even the training mode don't work as expected (hit, hurt and pushboxes disappeared) and nothing was logged into the .txt

**Comment by JordanAnthonyKing** [comment]
I'll try and give it a go over the weekend, something could've changed behind the scenes ¯\_(ツ)_/¯
